The program above tries to access the element in the eleventh index of an array that only has a length of ten and a max index of nine. As expected, the exception is thrown and the program stops executing due to the error. Let's change the number in the program to a number that should not trigger the error to check if our exception statement works as intended.

